piranha/urls.py

```diff
@@ -15,15 +15,15 @@
 """
 from django.contrib import admin
 from django.urls import path, include
-from django.views.decorators.csrf import csrf_exempt
 
 from piranha.views import index
 from seaport.views import PrivateGraphQLView
 
 urlpatterns = [
     path('', include('social_django.urls', namespace='social')),
+    path("api/graphql", PrivateGraphQLView.as_view(graphiql=True)),
+    # FIXME: Don't leave csrf_exempt here (something like https://docs.djangoproject.com/en/3.0/ref/csrf/#setting-the-token-on-the-ajax-request)
```

Comment:
```suggestion
```
You can remove this comment; I wrote it early on but I later added the CSRF middleware in the frontend to handle this properly